src/fee_lab_to_nwb/scherrer_ophys/convert_session.py

```diff
@@ -3,30 +3,34 @@
 from datetime import datetime
 from zoneinfo import ZoneInfo
 
+from natsort import natsorted
 from neuroconv.utils import load_dict_from_file, dict_deep_update
 
 from fee_lab_to_nwb.scherrer_ophys import ScherrerOphysNWBConverter
 from utils import get_timestamps_from_csv
 
-# The base folder path for ophys data
-ophys_dataset_path = Path("../../scherrer_ophys_data/")
-ophys_dataset_timestamp = "2021-06-03T11_46_29"
-# The name of the NWB file
-nwb_file_name = ophys_dataset_path.stem + "_" + ophys_dataset_timestamp
+# The base folder path for the calcium imaging data
+ophys_folder_path = Path("/Volumes/t7-ssd/fee-lab-to-nwb/ophys")
+# The timestamp for the recording
+ophys_dataset_timestamp = "2021-07-26T13_50_50"
 
-
-behavior_data_file_path = Path(f"./home_pos-speed-in_{ophys_dataset_timestamp}.csv")
-behavior_movie_file_path = Path(f"./home_arena_{ophys_dataset_timestamp}.avi")
-# Add a description for the behavior video
+# The file path to the behavior movie file
+behavior_movie_file_path = ophys_folder_path / f"./home_arena_{ophys_dataset_timestamp}.avi"
```

Comment:
```suggestion
behavior_movie_file_path = ophys_folder_path / f"home_arena_{ophys_dataset_timestamp}.avi"
```

The `/` operator on the `pathlib.Path` object is sufficient for specifying relative locations

Reply:
Sure, thank you for noticing this!